frontend/lib/laletterbuilder/letter-builder/my-letters.tsx

```diff
@@ -74,6 +39,55 @@ const MyLettersContent: React.FC = (props) => {
   );
 };
 
+const CreateOrContinueLetter: React.FC = (props) => {
+  const { session } = useContext(AppContext);
+
+  return (
+    <SessionUpdatingFormSubmitter
+      mutation={LaLetterBuilderCreateLetterMutation}
+      initialState={{}}
+      onSuccessRedirect={
+        LaLetterBuilderRouteInfo.locale.habitability.issues.prefix
+      }
+    >
+      {(sessionCtx) => (
+        <>
+          <div
+            className="my-letters-box"
+            // Hidden if there is a letter in progress
+            hidden={!!session.hasHabitabilityLetterInProgress}
```

Comment:
does using a ternary inside the form submitter (vs this hidden attribute) also cause the issue?

Reply:
That seems safer anyways, since hidden elements can be un-hidden from the browser